Misc/Set-BcContainerFeatureKeys.ps1

```diff
@@ -68,20 +82,37 @@ try {
                         $enabled = -1
                         Write-Host "WARNING: Unknown value ($enabledStr) for feature key $featureKey"
                     }
-                    if ($enabled -ne -1) {
+                    
+                    # Test if feature which has to be updated is available in table "Feature Data Update Status$63ca2fa4-4f03-4f2b-a480-172fef340d3f"
+                    $FeatureExistsInDestination = Invoke-Sqlcmd -Database $databaseName -Query $("SELECT COUNT(*) FROM [dbo].[Feature Data Update Status"+'$'+"63ca2fa4-4f03-4f2b-a480-172fef340d3f] where [Feature Key] = '$featureKey'")
+
+                    if(($FeatureExistsInDestination[0].ToString()) -eq "0")
+                    {

```

Comment:
The style in containerhelper all over is to place the { on the same line as the if statement - I know it is religion, but that is how it is all over and easier to read for me.

Reply:
sure...all adjusted